libtraceevent/parse-filter.c

```diff
@@ -1624,9 +1624,11 @@ int pevent_filter_clear_trivial(struct event_filter *filter,
 		case FILTER_TRIVIAL_FALSE:
 			if (filter_type->filter->boolean.value)
 				continue;
+			/* fall through */
 		case FILTER_TRIVIAL_TRUE:
 			if (!filter_type->filter->boolean.value)
 				continue;
+			/* fall through */
```

Comment:
@namhyung I sent [the patch](https://lkml.org/lkml/2017/4/26/393) and then Steven Rostedt told me he take this patch
but his remote repository isn't updated yet.

Apply the break version into this PR ? or wait for his upstream ?